CMake/External_Ceres.cmake

```diff
@@ -26,6 +26,13 @@ else()
   list(APPEND Ceres_EXTRA_BUILD_FLAGS -DMINIGLOG:BOOL=ON)
 endif()
 
+if (fletch_ENABLE_GFlags)
+  list(APPEND Ceres_DEPENDS GFlags)
+  list(APPEND Ceres_EXTRA_BUILD_FLAGS
+         -Dgflags_DIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/cmake/gflags
```

Comment:
Right, I just saw that too. It might be nice to add a GFlags_DIR to the
External_GFlags.cmake file and use that. It is arguable that it's out of
scope for this branch though, so either way is ok.

If you don't change anything before the dashboard builds finish I will
merge and look into that at a later time.